src/settings/toml/deploy_target.rs

```diff
@@ -25,22 +21,36 @@ impl RouteConfig {
         }
     }
 
+    pub fn routes_defined(&self) -> bool {
+        if let Some(pattern) = &self.route {
+            !pattern.is_empty() || self.routes.is_some() // this is all so messy because of deserializer
```

Comment:
i can check first if self.routes is present which would be better, but i do still need to get the pattern out of self.route and see if it is empty because (today) we allow you to have `route = ""` in your toml.